test/pi_patch.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2021, Pyronear contributors.
+
+# This program is licensed under the GNU Affero General Public License version 3.
+# See LICENSE or go to <https://www.gnu.org/licenses/agpl-3.0.txt> for full license details.
+
+import sys
+import fake_gpiozero
```

Comment:
Does `fake_gpiozero` should be added to the dev requirements ?

Reply:
no we do not whant to use it in the real package, it's just for testing on github

Reply:
moreover it's not a package so can't be installed ;)

Reply:
ho okay! So how to install it ?

Reply:
okay my bad I just get the tricks ^^Nice